ansible/group_vars/fanout/secrets.yml

```diff
@@ -1,6 +1,16 @@
+# Please update the actual username and password according to your lab configuration
+
 ansible_ssh_user: user
 ansible_ssh_pass: password
 fanout_mlnx_user: admin
 fanout_mlnx_password: admin
 fanout_sonic_user: admin
 fanout_sonic_password: password
+
```

Comment:
our vault based secrets file is downloaded to group_vars/all/secrets.yml. It has section for each inventory. We might end up managing two sets of password collections with the code, is this a concern?